### PR TITLE
Feat: added guest user view of passport

### LINKED ISSUE #3331: Create passport view for guest users
Guest users won't have a detailed passport, hence the need of creating a panel variation to support this:

Screenshot 2022-10-26 at 10 29 23

Acceptance criteria:
- [x] Guest users inspected will show this panel
- [x] Nothing will be displayed a part from the name and description

Technical assessment: https://www.notion.so/decentraland/RFC-Technical-Assessment-extended-passport-WIP-2bb32a1811114313b96e74a4efe3b855#5b5e0f6c92be4002820a6f0a484ef42d
Figma design: https://www.figma.com/file/iPUCfxQLHKXbLYqPlHZcg2/Passport-%2F-Profile?node-id=1957%3A6860

### PR DESCRIPTION
**Pull request by davidejensen**
## What does this PR change?

Fix #3331
Adds the guest view structure of the new passport

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/guest-passport&ENABLE_new_avatar_flow=
2. Open the passport of a regular user and verify it appears like this:
Screenshot 2022-11-18 at 10 19 47
3. Open the passport of a guest user and verify it appears like this:
Screenshot 2022-11-18 at 10 19 39

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using NSubstitute;
+using NUnit.Framework;
+using DCL.Social.Passports;
+using DCL;
+using SocialFeaturesAnalytics;
+
+public class PassportHUDControllerShould
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:10]: missing namespace

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    {
+        controller.Dispose();
+    }
+
+    [Test]
+    public void SetVisibilityTrueCorrectly()
+    {
+        controller.SetVisibility(true);
+
+        Assert.IsTrue(view.gameObject.activeSelf);
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:71]: i would recommend you to mock the view. You are doing an integration test if you use the real view and will bring extra complexity. In this case, as an example, you could assert: `view.Received(1).Show()`

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    [Test]
+    public void SetVisibilityFalseCorrectly()
+    {
+        controller.SetVisibility(false);
+
+        Assert.IsFalse(view.gameObject.activeSelf);
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:79]: you should do this kind of assertions in the tests of the view, not the controller

**Review by lorux0 — CHANGES_REQUESTED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    private PassportNavigationComponentController passportNavigationController;
+    private StringVariable currentPlayerInfoCardId;
+    private IUserProfileBridge userProfileBridge;
+    private ISocialAnalytics socialAnalytics;
+    private DataStore dataStore;
+    private IProfanityFilter profanityFilter;
+    private IFriendsController friendsController;
+
+    [SetUp]
+    public void SetUp()
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:25]: is missing many test cases of the class, are you planning to do it later?

Comment (davidejensen) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:25]: yes :)

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs (hunk 1 above)
Comment (davidejensen) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs:10]: done

**Review by lorux0 — APPROVED** [review]

**Pull request merged**